File: neoforge/src/main/java/city/sdy623/kitsuneime/neoforge/mixin/MixinKeyboardHandler.java
package city.sdy623.kitsuneime.neoforge.mixin;

import net.minecraft.client.KeyboardHandler;
import org.spongepowered.asm.mixin.Mixin;
import org.spongepowered.asm.mixin.gen.Invoker;

@Mixin(KeyboardHandler.class)
public interface MixinKeyboardHandler {

    @Invoker("charTyped")
    void invokeCharTyped(long window, int codePoint, int modifiers);
}

File: neoforge/src/main/kotlin/city/windmill/ingameime/neoforge/mixin/MixinKeyboardHandler.kt
package city.windmill.ingameime.neoforge.mixin

import net.minecraft.client.KeyboardHandler
import org.spongepowered.asm.mixin.Mixin
import org.spongepowered.asm.mixin.gen.Invoker

@Mixin(KeyboardHandler::class)
interface MixinKeyboardHandler {

    @Invoker("charTyped")
    fun invokeCharTyped(window: Long, codePoint: Int, modifiers: Int)
}


